src/dataset.jl

```diff
@@ -172,16 +172,132 @@ end
     write(dataset::AbstractDataset, filename::AbstractString; kwargs...)
 
 Writes the dataset to the designated filename.
+
+For the keyword arguments for vector datasets, consult the documentation of `writelayers` 
+and for raster datasets the docs for `writerasters`.
 """
 function write(
```

Comment:
I've added `writerasters` (1:1 copy of this write) and `writelayers`. Therefore `write` could be deprecated, but I didn't add the macro yet.